docs/lifecycle.md

```diff
@@ -0,0 +1,103 @@
+# Life Cycle Analysis for Advertising
+
+Life cycle assessment or LCA (also known as life cycle analysis) is a methodology for assessing environmental impacts associated with all the stages of the life cycle of a commercial product, process, or service. For instance, in the case of a manufactured product, environmental impacts are assessed from raw material extraction and processing (cradle), through the product's manufacture, distribution and use, to the recycling or final disposal of the materials composing it (grave). (from ["Life-Cycle Assessment"](https://en.wikipedia.org/wiki/Life-cycle_assessment) - Wikipedia).
+
+From [Wikipedia](https://en.wikipedia.org/wiki/File:Life_cycle_analysis_and_GHG_carbon_accounting.jpg), an LCA diagram with GHG phases along the right side
+![Example of an LCA with GHG emissions](images/lca_ghg.jpeg)
+
+## Life Cycle Assessment of an Advertisement
+
+To "manufacture" an ad impression, our raw materials are media (content), creative (the ad itself), and human attention (via ears and/or eyes). Different channels will have different combinations of these ingredients. For instance, a billboard often doesn't have content, just the ad. However, a billboard will have a disposal component when the vinyl is removed or recycled.
+
+The life cycle of an ad has five major components:
+
+1. Producing the creative
+2. Producing the content
+3. Selecting which ad to display
+4. Consumption of the media and creative, including disposal
+5. Consumer behavior after consuming the ad
+
+![Lifecycle of an ad](images/lca_ad.png)
+
+Note that this project *does not* recommend a life cycle boundary; rather, it produces outputs that can be included or excluded based on the selected life cycle span.
+
+## Methodologies
+
+There are three well-documented methodologies that we have seen produced by the industry in addition to this project:
+
+- [DIMPACT](https://dimpact.org/methodology)
+- [SRI](https://www.sri-france.org/wp-content/uploads/2021/11/SRI_Calculating-the-carbon-footprint_VF.pdf)
+- [GroupM / EY](https://www.groupm.com/media-decarbonization-framework-groupm/)
+
+We also include [Scope3](https://scope3.com) as a vendor built on top of this project to compare coverage to these methodologies.
+
+### Life cycle boundaries by methodology
+
+| Topic | DIMPACT | SRI | GroupM / EY | Scope3
+|---| --- | --- | --- | ---
+| Includes creative production | No | No | Yes | No
+| Includes content production | Partial (1) | No | Yes | Partial
```

Comment:
i liked how this partial had a reference to explain. should the other `Partials` throughout this document be accompanied by a footnote?